CMakeLists.txt

```diff
@@ -30,20 +30,14 @@ if(WITH_API)
 endif()
 
 find_package(MPI)
-if(MPI_Fortran_FOUND)
-  option(WITH_MPI "Whether DFTB+ should support MPI-parallelism" TRUE)
-else()
-  option(WITH_MPI "Whether DFTB+ should support MPI-parallelism" FALSE)
-endif()
 if(WITH_MPI AND NOT MPI_FORTRAN_FOUND)
   message(FATAL_ERROR "Compiler ${CMAKE_Fortran_COMPILER} is not MPI capable but is specified for "
     "a WITH_MPI=TRUE build")
 elseif(NOT WITH_MPI AND MPI_FORTRAN_FOUND)
   message(WARNING "MPI enabled compiler ${CMAKE_Fortran_COMPILER} is specified for a build where "
-    "WITH_MPI=FALSE")
+    "WITH_MPI=FALSE. Your build will NOT be MPI-parallelised.")
```

Comment:
Might be worth giving a hint as to how to build with MPI. Something like:
MPI capable compiler found but your build will NOT be MPI-parallelised. To enable this set WITH_MPI to TRUE

Reply:
:+1:  :heavy_check_mark: